app/vmui/packages/vmui/src/components/ExploreAlerts/Rule/index.tsx

```diff
@@ -0,0 +1,163 @@
+                  )}
+                </td>
+              </tr>
+              <tr>
+                <td className="col-small">Duration</td>
+                <td>{dayjs.duration(rule.duration, "seconds").humanize()}</td>
+              </tr>
+              <tr>
+                <td className="col-small">Last evaluation</td>
+                <td>{dayjs().to(dayjs(rule.lastEvaluation))}</td>
+              </tr>
+              {rule.lastError && (
+                <tr>
+                  <td className="col-small">Last error</td>
+                  <td>{rule.lastError}</td>
+                </tr>
+              )}
+              {rule.labels && (
+                <tr>
+                  <td className="col-small">Labels</td>
+                  <td>
+                    <div className="badge-container">
+                      {Object.entries(rule.labels).map(([name, value]) => (
+                        <span
+                          key={name}
+                          className="badge"
+                        >{`${name}: ${value}`}</span>
+                      ))}
+                    </div>
+                  </td>
+                </tr>
+              )}
+              {rule.annotations && (
+                <tr>
+                  <td colSpan="2">Annotations</td>
+                </tr>
+              )}
+              {Object.entries(rule.annotations || {}).map(([name, value]) => (
+                <tr key={name}>
+                  <td className="col-small">{name}</td>
+                  <td>{value}</td>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+          {rule.alerts && (
+            <>
+              <span className="title">Alerts</span>
+              <table>
+                <thead>
+                  <tr>
+                    <th className="col-small">Active since</th>
+                    <th className="col-small">Value</th>
+                    <th>Labels</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  {rule.alerts.map(alert => (
+                    <tr
+                      id={`alert-${alert.id}`}
+                      key={alert.id}
+                      onClick={openLink(alert.sourceLink)}
+                    >
+                      <td className="col-small">{dayjs().to(dayjs(alert.activeAt))}</td>
+                      <td className="col-small">
+                        <div className={`badge ${alert.state}`}>{alert.value}</div>
+                      </td>
+                      <td className="badge-container">
+                        {Object.entries(alert.labels || {}).map(([name, value]) => (
+                          <span
+                            key={name}
+                            className="badge"
+                          >{`${name}: ${value}`}</span>
+                        ))}
+                      </td>
+                    </tr>
+                  ))}
+                </tbody>
+              </table>
```

Comment:
Is it possible to use here `Table` component from [src/components/Table/Table.tsx](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/777f05efcc3ef3440c8821861dbc91357dd9c586/app/vmui/packages/vmui/src/components/Table/Table.tsx)?

Reply:
unfortunately it's not, visualisation is quite different